cmd/main.go

```diff
@@ -73,6 +78,24 @@ func main() {
 	logger := controller.NewProductionGRPCLogger(bufferedGrpcSyncer)
 	defer logger.Sync() //nolint:errcheck
 
+	// Convert zap.Logger to logr.Logger
+	logrLogger := zapr.NewLogger(logger)
+
+	// Set logrLogger as the global logger for klog
+	klog.SetLoggerWithOptions(logrLogger)
+
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		klog.Error(err, "Failed to load in-cluster config")
+		return
+	}
+
+	clientset, err := kubernetes.NewForConfig(config)
```

Comment:
Why duplicate the logic from `streams.go`?
Why would you need another client set?
https://github.com/illumio/cloud-operator/blob/9d48abbc5f565a9be2a1d8d9787ad291b9d3b0dd/internal/controller/streams.go#L172